org.kde.kompare.json

```diff
@@ -9,14 +9,12 @@
         "--share=ipc",
         "--socket=fallback-x11",
         "--socket=wayland",
-        "--device=dri"
+        "--device=dri",
+        "--system-talk-name=org.freedesktop.UDisks2"
     ],
     "modules": [
         {
             "name": "libkomparediff2",
-            "config-opts": [
```

Comment:
why remove this?

Reply:
https://buildbot.flathub.org/#/builders/8/builds/13186/steps/6/logs/stdio

> CMake Warning:
  Manually-specified variables were not used by the project:
    ENABLE_TESTING